main/src/ca/uwaterloo/flix/api/Bootstrap.scala

```diff
@@ -621,4 +629,68 @@ class Bootstrap(val projectPath: Path, apiKey: Option[String]) {
         }
     }
   }
+
+  /**
+    * Package the current project and release it on GitHub.
+    */
+  def release(flix: Flix): Validation[Unit, BootstrapError] = {
+    val formatter = flix.getFormatter
+
+    // Ensure that we have a manifest
+    val manifest = optManifest match {
+      case Some(m) => m
+      case None => return Validation.toHardFailure(BootstrapError.ReleaseError(ReleaseError.MissingManifest))
+    }
+
+    // Check if `github` option is present
+    val githubRepo = manifest.repository match {
+      case Some(r) => r
+      case None =>
+        return Validation.toHardFailure(BootstrapError.ReleaseError(ReleaseError.NoLinkedProject))
+    }
+
+    // Check if `--github-key` option is present
+    val githubKey = flix.options.githubKey match {
+      case Some(k) => k
+      case None =>
+        return Validation.toHardFailure(BootstrapError.ReleaseError(ReleaseError.MissingApiKey))
+    }
+
+    if (!flix.options.assumeYes) {
+      // Ask for confirmation
+      print(s"Release ${formatter.blue(s"github:$githubRepo")} ${formatter.yellow(s"v${manifest.version}")}? [y/N]: ")
+      val response = readLine()
+      response.toLowerCase match {
+        case "y" => // Continue
+        case _ => return Validation.toHardFailure(BootstrapError.ReleaseError(ReleaseError.Cancelled))
+      }
+    }
+
+    // Build artifacts
+    println("Building project...")
+    val buildResult = buildPkg()
+    buildResult match {
+      case Validation.Success(_) => // Continue
+      case Validation.SoftFailure(_, e) => return Validation.SoftFailure((), e)
+      case Validation.HardFailure(e) => return Validation.HardFailure(e)
+    }
+
+    // Publish to GitHub
+    println("Publishing new release...")
+    val artifacts = Files.list(getArtifactDirectory(projectPath)).iterator().asScala.toList
```

Comment:
Add a comment that after a build artifacts should contain `package.fpkg` and `flix.toml`.

Reply:
But still. This seems lazy and dangerous. Why not just make a list with exactly those two files?

Reply:
Is there an important distinction between the flix.toml in the project's root and the one in the artifacts directory?